src/main/java/duke/Event.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDate time;
+
+    Event(String description, String time) {
+        super(description);
+        this.time = LocalDate.parse(time);
+    }
```

Comment:
While `time` is an appropriate attribute name, the textbook mentions that a name could explain the named entity accurately and to a sufficient level of detail. Perhaps a name like `timeOfEvent` or `eventTime` would be more clear to readers?